doc/source/fault-tolerance.rst

```diff
@@ -58,3 +58,9 @@ Lost Objects
    evicted, and is later needed, Ray will not reconstruct this object.
 2. If an object is constructed by an actor method, is then evicted, and is later
    needed, Ray will not reconstruct this object.
+
+Actor Reconstruction
```

Comment:
I'm not sure if this really belongs under "Unhandled Failures". Maybe it should be under "Actors"?

Reply:
Oh, I thought it would go better here since the other limitation for actors (no reconstruction for evicted objects) is also listed here, in the above subsection. Should I move it?

Reply:
Actually, why don't we leave it here and just change the name "Unhandled Failures" to "Current Limitations" or something like that?